scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -0,0 +1,76 @@
+package scalafix.internal.config
+
+import metaconfig.ConfDecoder
+import metaconfig.Configured.{NotOk, Ok}
+import org.langmeta.Symbol
+
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+import scalafix.internal.util._
+
+sealed trait UnlessMode
+case object UnlessInside extends UnlessMode
+case object UnlessOutside extends UnlessMode
+
+case class UnlessConfig(
+    mode: UnlessMode,
+    block: Symbol.Global,
+    symbol: Symbol.Global,
+    message: Option[String])
+
+object UnlessConfig {
+  implicit val decoder: ConfDecoder[UnlessConfig] =
+    ConfDecoder.instanceF[UnlessConfig] { c =>
+      (
+        (c.get[String]("mode") match {
+          case Ok("inside") => Ok(UnlessInside)
+          case Ok("outside") => Ok(UnlessOutside)
+          case _ => Ok(UnlessInside)
+        }) |@|
+          c.get[Symbol.Global]("block") |@|
+          c.get[Symbol.Global]("symbol") |@|
+          (c.get[String]("message") match {
```

Comment:
There is now `getOption[String]` after the upgrade to metaconfig v0.5.4 a couple days ago